pocket_tts/__init__.py

```
@@ -3,7 +3,10 @@
 
 beartype_this_package(conf=BeartypeConf(is_color=False))
 
-from pocket_tts.models.tts_model import TTSModel  # noqa: E402
+from pocket_tts.models.tts_model import (  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)